### ISSUE TITLE
Changes necessary for OSX

### ISSUE THREAD

**Issue by csytsma** [dbaron/osm-satellite-scripts#1]
1. 'mktemp' for OSX doesn't have the '-p' parameter, so I commented it out.
2. So I also removed any steps that involved moving into and out of the tmp directory
3. I had to change all of the file references that started with '../' to a single period './' 

I'd probably also recommend being more explicit about the dependencies.  For OSX, I had to:
`brew install imagemagick`
`brew install libgeotiff`

I didn't have to edit the ImageMagick policy.xml file. I actually did edit it, but realized they were still commented out, and it appeared to work. On OSX, the file is located at:
`/usr/local/etc/ImageMagick-7/policy.xml`

I think during this command `convert B0{4,3,2}-8bit.tif -combine RGB.tif || exit $?`, I get about a dozen of the following Error/Warnings. The tag/hex number is different for each:
`convert: Unknown field with tag 33550 (0x830e) encountered. `TIFFReadDirectory' @ warning/tiff.c/TIFFWarnings/997`
It doesn't appear to have an effect on the finished file though.

Thanks for putting this together. I had no idea it was possible to freely pull current satellite photos. Let me know if any questions, or need me to test the script again.

-Cory

**Comment by dbaron** [comment]
I think I fixed the `mktemp` usage to be more portable based on the [`mktemp` man page on Darwin](http://polarhome.com/service/man/?qf=mktemp&tf=2&of=Darwin&sf=).  If you want to test that it works now, that would be appreciated although not critical.  (I've checked that it still works on Linux.)

The warnings are definitely normal; I've always assumed it wasn't a problem since TIFF is a very extensible format, and it just seems like the program is warning that there are parts it doesn't understand.

I still need to look into the documentation issues another day...
